packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { ActionSheet, Text, TextField, TextFieldWithSteppers, ToggleSwitch } from '@deriv-com/quill-ui';
+import { Localize, localize } from '@deriv/translations';
+import RiskManagementInfoModal from '../RiskManagementInfoModal';
+import DealCancellationRemainingTime from '../DealCancellationRemainingTime/deal-cancellation-remaining-time';
+import { observer } from '@deriv/stores';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import { CONTRACT_TYPES, isAccumulatorContract, isValidToCancel } from '@deriv/shared';
+
+type RiskManagementItemProps = {
+    label: React.ReactNode;
+    modal_body_content: React.ReactNode;
+    is_deal_cancellation?: boolean;
+    value?: number | null;
+    type?: string;
+};
+
+const RiskManagementItem = observer(
+    ({ label, modal_body_content, is_deal_cancellation = false, value, type }: RiskManagementItemProps) => {
+        const [toggle, setToggle] = React.useState(Boolean(value));
```

Comment:
What is `toggle`? I mean it's used as boolean for `ToggleSwitch`, maybe we can rename it to `isOn`? Or something more boolean-like